app/components/ui/files/publish-modal/component.js

```diff
@@ -6,18 +6,23 @@ import config from '../../../../config/environment';
 export default Ember.Component.extend({
     authRequest: Ember.inject.service(),
     internalState: Ember.inject.service(),
+    // Controls the state of the publish button
     enablePublish: true,
+    // Set the default repository to dev
     selectedRepository: 'Development',
-    taleId: '5b5b9a4d9de633056d436adf',
     // Flag set to show the spinner
     publishing: false,
+    // Set when publishing has finished
     publishingFinish: false,
     // An array that holds a pair, (fileName, fileSize)
     fileList: [],
     // Holds an array of objects that cannot be excluded from publishing
     immutaleFile: ['tale.yaml', 'environment.zip'],
+    // A map that connects the repository dropdown to a url
     repositoryMapping: {'Development': 'https://dev.nceas.ucsb.edu/knb/d1/mn/'},
+    // The url for the published tale. This is set after publication succeeds
     packageUrl: '',
+    // The jwt token that allows the user to interact with DataONE
```

Comment:
@ThomasThelen I would have named this token variable differently, just in case we need to add another token for interacting with a different data source or system.